documentation/book/assembly-upgrade-resources.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-upgrade-resources-{context}']
+= {ProductName} resource upgrades
+For this release of {ProductName}, resources that use the API version `{KafkaApiVersionPrev}` must be updated to use `{KafkaApiVersion}`.
+
+The `{KafkaApiVersionPrev}` API version has been deprecated for this release.
+
+IMPORTANT: The upgrade of resources _must_ be performed before xref:assembly-upgrade-{context}[], so the Cluster Operator can understand the CRDs.
```

Comment:
This is not true. The Strimzi 0.12.0 CO will understand the v1alpha1 resources just fine, and upgrading the resources must happen after the upgrade of the CO, because the 0.11.x CO won't understand the v1beta1 resources.